packages/xo-web/src/xo-app/logs/backup-ng/log-alert-header.js

```diff
@@ -91,7 +108,8 @@ export default decorate([
         </Tooltip>
         {CAN_REPORT_BUG && (
           <ReportBugButton
-            message={`\`\`\`json\n${state.formattedLog}\n\`\`\``}
+            files={state.files}
+            message={state.message}
```

Comment:
deconstruct `state`

Reply:
Why it's needed her?

Reply:
to not access to `state` multiple times but it's not too essential. if it's OK for you I have no problem ^_^

Reply:
It's used one time.